setup.cfg

```diff
@@ -28,13 +29,18 @@ zip_safe = False
 install_requires =
     invenio-base>=1.2.5
     invenio-i18n>=1.3.1
+    invenio-records-resources>=1.0.0,<2.0.0
 
 [options.extras_require]
 tests =
     invenio-app>=1.3.4,<2.0.0
     pytest-invenio>=2.1.0,<3.0.0
     pytest-black>=0.3.0
     sphinx>=4.5.0
+elasticsearch7 =
+    invenio-search[elasticsearch7]>=2.1.0,<3.0.0
+opensearch2 =
+    invenio-search[opensearch2]>=2.1.0,<3.0.0
```

Comment:
```suggestion
```

Same as above, we'll probably won't need these here. Remember to check the GitHub CI tests matrix as well and remove anything relevant to search from there.

Reply:
Added this because of failing testcases as records-resources imports from invenio-search (see comment above). Without records-resources, this is also not needed